drivers/segger/sysview.c

```diff
@@ -371,7 +385,7 @@ void sched_note_irqhandler(int irq, FAR void *handler, bool enter)
 #endif
 
 #ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
-void sched_note_syscall_enter(int nr, int argc, ...)
+static void sysview_syscall_enter(FAR struct note_driver_s *drv, int nr)
 {
   nr -= CONFIG_SYS_RESERVED;
 
```

Comment:
remove sysview_isenabled_syscall and NOTE_FILTER_SYSCALLMASK_ISSET, already check in common code

Reply:
done